cmd/agent/common/import.go

```diff
@@ -154,6 +124,36 @@ func ImportConfig(oldConfigDir string, newConfigDir string, force bool) error {
 		)
 	}
 
+	// backup the original datadog.yaml to datadog.yaml.bak
```

Comment:
I am wondering what is the reason to move this code?

Reply:
It was because some to-be-converted configuration values were added to the configuration object after it was dump to the new `datadog.yaml`, so they were missing from the new configuration file.